client/src/context/LocationContext.js

```diff
@@ -1,56 +1,65 @@
 import createDataContext from "./createDataContext";
 
+const actionTypes = {
```

Comment:
Yes this is perfect. And remember you can use this everywhere you feel like you will be running into spelling issue. For example, the names of screens in your screen navigator.

Reply:
And you keep them inside a separate folder and export it. So you can import them anywhere you need to reference them.